packages/trader/src/Modules/Trading/Components/Elements/Multiplier/__tests__/risk-management-info.spec.tsx

```diff
@@ -0,0 +1,117 @@
+            </div>
+        ) : null
+    )
+);
+
+describe('<RiskManagementInfo />', () => {
+    const mockRiskManagementInfo = (mocked_store: TCoreStores) => {
+        return (
+            <TraderProviders store={mocked_store}>
+                <RiskManagementInfo />
+            </TraderProviders>
+        );
+    };
+
+    it('should render only risk management block if has_take_profit, has_stop_loss, has_cancellation are equal to false', () => {
+        const mock_root_store = mockStore(default_mock_store);
+        render(mockRiskManagementInfo(mock_root_store));
+
+        expect(screen.getByText(risk_management)).toBeInTheDocument();
+        expect(screen.queryByText(take_profit)).not.toBeInTheDocument();
+        expect(screen.queryByText(stop_loss)).not.toBeInTheDocument();
+        expect(screen.queryByText(deal_cancellation)).not.toBeInTheDocument();
+        expect(screen.queryByText(risk_management_dialog)).not.toBeInTheDocument();
+    });
+    it('should render has_take_profit, has_stop_loss, has_cancellation blocks if proper values are equal to true and not render risk management block ', () => {
+        default_mock_store.modules = {
+            trade: {
+                currency: 'USD',
+                stop_loss: 10,
+                take_profit: 30,
+                has_take_profit: true,
+                has_stop_loss: true,
+                cancellation_duration: '60m',
+                cancellation_range_list: [{ value: '60m', text: 'mocked text' }],
+                has_cancellation: true,
+            },
+        };
+        const mock_root_store = mockStore(default_mock_store);
+        render(mockRiskManagementInfo(mock_root_store));
+
+        expect(screen.getByText(take_profit)).toBeInTheDocument();
+        expect(screen.getByText(/30.00 usd/i)).toBeInTheDocument();
+        expect(screen.getByText(stop_loss)).toBeInTheDocument();
+        expect(screen.getByText(/10.00 usd/i)).toBeInTheDocument();
+        expect(screen.getByText(deal_cancellation)).toBeInTheDocument();
+        expect(screen.getByText(/mocked text/i)).toBeInTheDocument();
+        expect(screen.queryByText(risk_management)).not.toBeInTheDocument();
+    });
+    it('should render <RiskManagementDialog /> if user clicked on one of the risk management blocks', () => {
+        const mock_root_store = mockStore(default_mock_store);
+        render(mockRiskManagementInfo(mock_root_store));
+
+        const risk_management_block = screen.getByText(take_profit);
+        userEvent.click(risk_management_block);
+
+        expect(screen.getByText(risk_management_dialog)).toBeInTheDocument();
+    });
+    it('should close <RiskManagementDialog /> if user clicked on close button', () => {
+        const mock_root_store = mockStore(default_mock_store);
+        render(mockRiskManagementInfo(mock_root_store));
+
+        const risk_management_block = screen.getByText(take_profit);
+        userEvent.click(risk_management_block);
```

Comment:
ditto

Reply:
Please, refer to https://github.com/binary-com/deriv-app/pull/9586/files#r1398838863